packages/modal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pcln-modal",
-  "version": "3.3.0",
+  "version": "3.4.0",
```

Comment:
Should we hold off on version bumps until the hackathon is over?

Reply:
No I think it's fine unless it's a major 😛

Reply:
Unless we want to group all our bumps into one

Reply:
I guess we also don't need to publish each time we bump the version